crates/storage/store/storage.rs

```diff
@@ -325,6 +325,24 @@ impl Store {
         Ok(txs_by_sender)
     }
 
+    /// Gets hashes from possible_hashes that are not already known in the mempool.
+    pub fn get_unknown_transactions(
```

Comment:
Nit:
get_unknown_transactions sounds a bit weird to me.
I think I would maybe call this 'filter_known_transactions', what do you think?

Reply:
Done